lib/traits/thermostat_widget.dart

```diff
@@ -86,8 +88,8 @@ class ThermostatWidget extends StatelessWidget with ToastNotifications {
         ),
         TemperatureRangeSlider(
           value,
-          temperatureRange.min,
-          temperatureRange.max,
+          temperatureRange?.min ?? 0,
```

Comment:
temperatureRange min is non nullable so I think it will be safe to put `!` here

Reply:
The temperature range itself is nullable according to the documentation

Reply:
I believe we resolved this in code sync this morning, correct?

Reply:
The question for a UI widget would be this and it's sort of separate. If there is no temp range because it can be null, we aren't supposed to be validating against anything, just setting the value. If all values of double are valid it's probably not good to set the slider min to `-double.maxFinite`. Would it be preferable that we would maybe show 100 degrees around the current value or something similar?